test/.gitignore

```diff
@@ -0,0 +1,11 @@
+/bins
+/fuzz/targets
+
+/t/io/.newfile
+/t/seek/.empty.tmp
```

Comment:
If `/t/` folder is dead, are those still relevant?

Reply:
No, removed.